code/modules/atmospherics/machinery/airalarm.dm

```diff
@@ -931,6 +933,8 @@
 			else
 				target_temperature = input_temperature
 
+		if("thermostat_state")
+			thermostat_state = text2num(params["val"])
```

Comment:
This can be streamlined a little by just making it a toggle rather than taking the value sent by the UI.
```suggestion
		if("thermostat_state")
			thermostat_state = !thermostat_state
```